frontend/src/components/PostAddingForm/PostAddingForm.jsx

```diff
@@ -69,7 +77,7 @@ const PostAddingForm = ({ formSubmit }) => {
   }
 
   return (
-    <Form onSubmit={handleSubmit(onSubmit)}>
+    <Form onSubmit={handleSubmit((data) => onSubmit(data))}>
```

Comment:
tu nie trzeba wywoływać :) Wystarczy przekazać funkcję :D
```suggestion
    <Form onSubmit={handleSubmit(onSubmit)}>
```